src/vs/code/browser/workbench/workbench-dev.html

```diff
@@ -41,6 +41,11 @@
 		Object.keys(self.webPackagePaths).map(function (key, index) {
 			self.webPackagePaths[key] = `${baseUrl}/remote/web/node_modules/${key}/${self.webPackagePaths[key]}`;
 		});
+
+		if (!navigator.language.startsWith('en')) {
+			self.webPackagePaths[`vs/workbench/workbench.web.main.nls.${navigator.language}`] = `{{WORKBENCH_NLS_BASE_URL}}/vs/code/browser/workbench/workbench.nls.${navigator.language}.js`;
```

Comment:
We might have other entry points that we'd need to cover here. Given our Slack discussion, maybe an easier path would be to have an anonymous `define` call inside `workbench.nls.LANG.js` and then the nls plugin can load it with any name it wants.